Source/Relativity.Test.Helpers/TestHelpersKepler/TestHelpersKepler.Services/TestHelpersModule/v1/TestHelpersService.cs

```diff
@@ -129,6 +129,41 @@ public async Task<FieldCountResponseModel> GetFieldCountAsync(int artifactId, in
 			return responseModel;
 		}
 
+		public async Task<GetFolderNameResponseModel> GetFolderNameAsync(int folderArtifactId, int workspaceId)
+		{
+			GetFolderNameResponseModel responseModel;
+			try
+			{
+				string folderName = await _helper.GetDBContext(workspaceId).ExecuteScalarAsync<string>(
+					new ContextQuery
+					{
+						SqlStatement = "select Name from folder where ArtifactID = @folderArtifactId",
```

Comment:
the SQL statement could be written better. we need to use [EDDSDBO].[Folder] for table name. 

And also add NOLOCK for query statements. You should be able to find an example for NOLOCK from SolutionSnapshot repo.

Reply:
Just copied this sql statement from what was previously there